decorator/decorator_resolver_test.go

```diff
@@ -91,7 +103,7 @@ func TestDecoratorResolver(t *testing.T) {
 				},
 				"root/main",
 			)
-			os.RemoveAll(root)
+			err = os.RemoveAll(root)
```

Comment:
No this was intentional - I want to remove the tomporary folder (and ignore any error), then check the error that was returned by `packages.Load`.

Perhaps change to:

```go
_ = os.RemoveAll(root) // ignore error
```